### PR TITLE
WPF NET Core, fix WPF renderer, fix Android X

### LINKED ISSUE #143: .NET Core WPF and GTK
It would be nice to add support for the same platforms as XF supports.

I see that mostly UWP and WPF are supported by community.
Are you ok with adding this new platforms? I can help you with it.

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
Fixes #143

### CODE REVIEW

File: example/Thewissen.PancakeViewSample.iOS/Thewissen.PancakeViewSample.iOS.csproj [hunk 1]
```diff
@@ -203,9 +203,15 @@
     <ImageAsset Include="Assets.xcassets\monkey.imageset\monkey.png">
       <Visible>false</Visible>
     </ImageAsset>
-    <ImageAsset Include="Assets.xcassets\xamalogo.imageset\xamalogo%403x.png" />
-    <ImageAsset Include="Assets.xcassets\xamalogo.imageset\xamalogo%402x.png" />
-    <ImageAsset Include="Assets.xcassets\xamalogo.imageset\xamalogo.png" />
+    <ImageAsset Include="Assets.xcassets\xamalogo.imageset\xamalogo%403x.png">
```
Comment (VladislavAntonyuk) [inline example/Thewissen.PancakeViewSample.iOS/Thewissen.PancakeViewSample.iOS.csproj:206]: Visual studio automatic changes

File: example/Thewissen.PancakeViewSample.Android/Thewissen.PancakeViewSample.Android.csproj [hunk 2]
```diff
@@ -73,6 +73,7 @@
         <PackageReference Include="Xamarin.Android.Support.Media.Compat">
             <Version>28.0.0.3</Version>
         </PackageReference>
+    </ItemGroup>
```
Comment (VladislavAntonyuk) [inline example/Thewissen.PancakeViewSample.Android/Thewissen.PancakeViewSample.Android.csproj:76]: fix project load failed

File: example/Thewissen.PancakeViewSample.WPF/Thewissen.PancakeViewSample.WPF.csproj [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace>Thewissen.PancakeViewSample.WPF</RootNamespace>
     <AssemblyName>Thewissen.PancakeViewSample.WPF</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (VladislavAntonyuk) [inline example/Thewissen.PancakeViewSample.WPF/Thewissen.PancakeViewSample.WPF.csproj:11]: XF WPF uses net 472 and .NET Core 3.1

File: src/Libs/gtk-sharp/README.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿https://github.com/mono/gtk-sharp
```
Comment (VladislavAntonyuk) [inline src/Libs/gtk-sharp/README.txt:1]: libs from XF sources

File: src/Xamarin.Forms.PancakeView.Multi/Platforms/Android/PancakeViewRenderer.cs [hunk 5]
```diff
@@ -6,7 +6,11 @@
 using Android.Graphics.Drawables;
 using Android.OS;
 using Android.Runtime;
+#if MONOANDROID10_0
```
Comment (VladislavAntonyuk) [inline src/Xamarin.Forms.PancakeView.Multi/Platforms/Android/PancakeViewRenderer.cs:9]: use Android X for MonoAndroid 10

File: src/Xamarin.Forms.PancakeView.Multi/Platforms/WPF/PancakeViewRenderer.cs [hunk 6]
```diff
@@ -1,8 +1,6 @@
-﻿using System;
-using System.ComponentModel;
+﻿using System.ComponentModel;
```
Comment (VladislavAntonyuk) [inline src/Xamarin.Forms.PancakeView.Multi/Platforms/WPF/PancakeViewRenderer.cs:1]: remove unused usings

File: src/Xamarin.Forms.PancakeView.Multi/Xamarin.Forms.PancakeView.Multi.csproj [hunk 7]
```diff
@@ -1,103 +1,146 @@
...
-        <Compile Include="Platforms\Android\**\*.cs" />
-        <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" />
-    </ItemGroup>
-    <ItemGroup Condition=" $(TargetFramework.StartsWith('xamarin.mac')) ">
-        <Compile Include="Platforms\macOS\**\*.cs" />
-    </ItemGroup>
-    <ItemGroup Condition=" $(TargetFramework.StartsWith('uap10.0')) ">
-        <Compile Include="Platforms\UWP\**\*.cs" />
-        <!--
+	<PropertyGroup>
```
Comment (VladislavAntonyuk) [inline src/Xamarin.Forms.PancakeView.Multi/Xamarin.Forms.PancakeView.Multi.csproj:3]: fixed alignment

**Comment by sthewissen** [comment]
Hey Vladislav, I plan to look at this sometime today. I also saw you added some AndroidX bits. I'm curious to see how/if those work, as I've had quite some issues with that in recent times. As I said, hopefully I'll get around to do some testing later today.

**Comment by VladislavAntonyuk** [comment]
I can try to update sample project to Android 10 or even 11.
It works on android 9 now

**Comment by VladislavAntonyuk** [comment]
it also fixes WPF renderer:
image

**Comment by VladislavAntonyuk** [comment]
Android 11:
image

File: example/Thewissen.PancakeViewSample.macOS/Thewissen.PancakeViewSample.macOS.csproj [hunk 8]
```diff
@@ -48,23 +53,16 @@
...
+    <HttpClientHandler>
+    </HttpClientHandler>
+    <XamMacArch>
+    </XamMacArch>
     <AOTMode>None</AOTMode>
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="Xamarin.Mac" />
```
Comment (VladislavAntonyuk) [inline example/Thewissen.PancakeViewSample.macOS/Thewissen.PancakeViewSample.macOS.csproj:65]: updated to PackageReference

File: src/Xamarin.Forms.PancakeView.Multi/Xamarin.Forms.PancakeView.Multi.csproj [hunk 9]
```diff
@@ -1,103 +1,144 @@
...
+		<TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.Mac20;tizen40</TargetFrameworks>
```
Comment (VladislavAntonyuk) [inline src/Xamarin.Forms.PancakeView.Multi/Xamarin.Forms.PancakeView.Multi.csproj:4]: I added MonoAndroid10.0 instead of replacing MonoAndroid9.0

**Comment by VladislavAntonyuk** [comment]
@sthewissen any updates?

**Comment by sthewissen** [comment]
Looking good from what I can tell. I couldn't run the UWP sample, not sure if you can? It complains on this line:

```
assembliesToInclude.Add(typeof(Xamarin.Forms.PancakeView.UWP.PancakeViewRenderer).GetTypeInfo().Assembly);
```

Claiming it can't find the UWP namespace. May also be an issue on my environment's end though, as I don't typically run any of the Windows side.

**Comment by VladislavAntonyuk** [comment]
Thank you for the response. Hope you are doing well.
I didn’t try UWP. I even didn’t touch it. I will take a look at it

**Comment by VladislavAntonyuk** [comment]
I'vr just checked it on UWP. it works. you probably need to uninstalled the previous version and rebuild the project

**Comment by sthewissen** [comment]
> Thank you for the response. Hope you are doing well.
> I didn’t try UWP. I even didn’t touch it. I will take a look at it

All is well, we just delivered a baby onto the world so its been a bit busy :)

**Review by sthewissen — APPROVED** [review]

**Pull request merged**